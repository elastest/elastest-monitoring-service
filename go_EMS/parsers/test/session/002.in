session fst := (begin => e.tag(#TJob) , end => e.tag(#SuT))
session snd := (begin => e.tag(#SuT) , end => e.tag(#TJob))
