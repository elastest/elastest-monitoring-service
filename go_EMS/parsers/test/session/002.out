There are 1 stampers
There are 0 sessions
There are 0 streams
There are 0 triggers
when true do #Tjob
